fil-proofs-tooling/src/bin/benchy/porep.rs

```diff
@@ -78,7 +78,9 @@ fn get_porep_config(sector_size: u64, api_version: ApiVersion, use_synthetic: bo
     // Replicate the staged sector, write the replica file to `sealed_path`.
     let mut config = PoRepConfig::new_groth16(sector_size, arbitrary_porep_id, api_version);
     if use_synthetic {
-        config.enable_feature(ApiFeature::SyntheticPoRep);
+        config
```

Comment:
We should be able to re-factor these config updates using `new_groth16_with_features` now

Reply:
That's a bigger refactor, hence I've opened a separate PR for it at https://github.com/filecoin-project/rust-fil-proofs/pull/1742.